ci/requirements_dev.txt

```diff
@@ -5,4 +5,4 @@ cython
 pytest>=3.1.0
 pytest-cov
 flake8
```

Comment:
revert this, s3fs is NOT a requirement for dev; we should be robust to not having this installed